tests/everflow/everflow_test_utilities.py

```diff
@@ -205,89 +190,60 @@ def get_port_info(in_port_list, out_port_list, out_port_ptf_id_list, out_port_la
-                "server_dest_ports_ptf_id": server_dest_ports_ptf_id,
-                "t1_ports": t1_ports,
-                DOWN_STREAM: {
-                    "src_port": t1_ports[0],
-                    "src_port_lag_name":t1_dest_lag_name[0],
-                    "src_port_ptf_id": str(mg_facts["minigraph_ptf_indices"][t1_ports[0]]),
-                    # Downstream traffic ingress from the first portchannel,
-                    # and mirror packet egress from other portchannels
-                    "dest_port": t1_dest_ports[1:],
-                    "dest_port_ptf_id": t1_dest_ports_ptf_id[1:],
-                    "dest_port_lag_name": t1_dest_lag_name[1:],
-                    "namespace": server_namespace
-                },
-                UP_STREAM: {
-                    "src_port": server_ports[0],
-                    "src_port_lag_name":"Not Applicable",
-                    "src_port_ptf_id": str(mg_facts["minigraph_ptf_indices"][server_ports[0]]),
-                    "dest_port": t1_dest_ports,
-                    "dest_port_ptf_id": t1_dest_ports_ptf_id,
-                    "dest_port_lag_name": t1_dest_lag_name,
-                    "namespace": t1_namespace
-                },
-            }
-        )
-    elif 't1' in topo:
-        # Downstream traffic (T1 -> T0)
-        tor_dest_ports = []
-        tor_dest_ports_ptf_id = []
-        tor_dest_lag_name = []
-        get_port_info(tor_ports, tor_dest_ports, tor_dest_ports_ptf_id, tor_dest_lag_name)
-        
-        # Upstream traffic (T0 -> T1)
-        spine_dest_ports = []
-        spine_dest_ports_ptf_id = []
-        spine_dest_lag_name = []
-        get_port_info(spine_ports, spine_dest_ports, spine_dest_ports_ptf_id, spine_dest_lag_name)
+    # Downstream traffic
+    downstream_dest_ports = []
+    downstream_dest_ports_ptf_id = []
+    downstream_dest_lag_name = None if topo_type in DOWNSTREAM_SERVER_TOPO else []
+    get_port_info(downstream_ports, downstream_dest_ports, downstream_dest_ports_ptf_id, downstream_dest_lag_name)
+
+    # Upstream traffic
+    upstream_dest_ports = []
+    upstream_dest_ports_ptf_id = []
+    upstream_dest_lag_name = []
+    get_port_info(upstream_ports, upstream_dest_ports, upstream_dest_ports_ptf_id, upstream_dest_lag_name)
+
+    setup_information.update(
+        {
+            "topo": topo_type,
+            DOWN_STREAM: {
+                "src_port": upstream_ports[0],
+                "src_port_lag_name": upstream_dest_lag_name[0],
+                "src_port_ptf_id": str(mg_facts["minigraph_ptf_indices"][upstream_ports[0]]),
+                # For T0 topo, downstream traffic ingress from the first portchannel,
+                # and mirror packet egress from other portchannels
+                "dest_port": upstream_dest_ports[1:] \
+                if topo_type in DOWNSTREAM_SERVER_TOPO else downstream_dest_ports,
+                "dest_port_ptf_id": upstream_dest_ports_ptf_id[1:] \
```

Comment:
I don't understand this. If the DUT's topology is T0, why the dst port is downstream_dest_ports (server facing port)?
In the original design, for T0 topology the testing traffic is ingressed from the first LAG, and the mirrored packet is egressed from the other LAG. Both src and dst ports are uplink ports.